### ISSUE TITLE
Ember modifier v3 depreciation

### ISSUE THREAD

**Issue by oliverlj** [lifeart/ember-ref-bucket#36]
Here is the migration guide for next ember modifier v4 : https://github.com/ember-modifier/ember-modifier/blob/v3/MIGRATIONS.md

```
testem.js:967 DEPRECATION: ember-modifier (in RefModifier at Error
    at new ClassBasedModifier (http://localhost:7357/assets/vendor.js:91247:228)
    at new RefModifier (http://localhost:7357/assets/vendor.js:92514:7)
    at ClassBasedModifierManager.createModifier (http://localhost:7357/assets/vendor.js:91062:24)
    at CustomModifierManager.create (http://localhost:7357/assets/vendor.js:37782:31)
    at Object.evaluate (http://localhost:7357/assets/vendor.js:45257:25)
    at AppendOpcodes.evaluate (http://localhost:7357/assets/vendor.js:43797:19)
    at LowLevelVM.evaluateSyscall (http://localhost:7357/assets/vendor.js:47663:22)
    at LowLevelVM.evaluateInner (http://localhost:7357/assets/vendor.js:47619:14)
    at LowLevelVM.evaluateOuter (http://localhost:7357/assets/vendor.js:47611:14)
    at VM.next (http://localhost:7357/assets/vendor.js:48622:24)): `willDestroy`, `isDestroyed`, and `isDestroyed` are deprecated. Use the corresponding API from '@ember/destroyable' instead. [deprecation id: ember-modifier.use-destroyables]
        at logDeprecationStackTrace (http://localhost:7357/assets/test-support.js:582:21)
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at raiseOnDeprecation (http://localhost:7357/assets/test-support.js:611:9)
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at http://localhost:7357/assets/test-support.js:6448:10
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at invoke (http://localhost:7357/assets/test-support.js:701:9)
        at deprecate (http://localhost:7357/assets/test-support.js:657:28)
        at new ClassBasedModifier (http://localhost:7357/assets/vendor.js:91247:160)
```

```
testem.js:967 DEPRECATION: ember-modifier (in RefModifier at Error
    at RefModifier.get [as args] (http://localhost:7357/assets/vendor.js:91437:155)
    at RefModifier.get name [as name] (http://localhost:7357/assets/vendor.js:92653:19)
    at new RefModifier (http://localhost:7357/assets/vendor.js:92516:42)
    at ClassBasedModifierManager.createModifier (http://localhost:7357/assets/vendor.js:91062:24)
    at CustomModifierManager.create (http://localhost:7357/assets/vendor.js:37782:31)
    at Object.evaluate (http://localhost:7357/assets/vendor.js:45257:25)
    at AppendOpcodes.evaluate (http://localhost:7357/assets/vendor.js:43797:19)
    at LowLevelVM.evaluateSyscall (http://localhost:7357/assets/vendor.js:47663:22)
    at LowLevelVM.evaluateInner (http://localhost:7357/assets/vendor.js:47619:14)
    at LowLevelVM.evaluateOuter (http://localhost:7357/assets/vendor.js:47611:14)): using `this.args` is deprecated. Access positional and named arguments directly in the `modify` hook instead. [deprecation id: ember-modifier.no-args-property]
        at logDeprecationStackTrace (http://localhost:7357/assets/test-support.js:582:21)
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at raiseOnDeprecation (http://localhost:7357/assets/test-support.js:611:9)
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at http://localhost:7357/assets/test-support.js:6448:10
        at HANDLERS.<computed> (http://localhost:7357/assets/test-support.js:689:9)
        at invoke (http://localhost:7357/assets/test-support.js:701:9)
        at deprecate (http://localhost:7357/assets/test-support.js:657:28)
        at RefModifier.get [as args] (http://localhost:7357/assets/vendor.js:91437:87)
```

**Comment by bwbuchanan** [comment]
Also need to add a dependency on `ember-auto-import` for compatibility with `ember-modifier` v4.0

**Comment by lifeart** [comment]
Thank you for report, should be solved with https://github.com/lifeart/ember-ref-bucket/pull/45

**Comment by lifeart** [comment]
fixed in v 5.0.0
